hoomd/md/AnisoPotentialPair.h

```diff
@@ -32,6 +32,8 @@
 
 #include <pybind11/pybind11.h>
 
+using namespace hoomd::detail;
```

Comment:
```suggestion
```
Add `detail::` where needed. `using` directives in header files will apply to all files that include it.